apps/api/src/service/email/layouts/templates/EmailInstitutionalRepReview.ts

```diff
@@ -0,0 +1,112 @@
+	} = getEmailConfig;
+
+	const template = `  
+            <mj-column css-class="section-wrapper">
+                <mj-text>
+                    Dear ${repName},
+                </mj-text>
+                <mj-text>
+                    We hope this message finds you well.
+                </mj-text>
+                <mj-text>
+                    A new PCGL <a href="" target="_blank" rel="nofollow">DACO application</a> has been submitted and is now awaiting your review. Please login to the DACO portal using your institutional email and review the application. Your timely input and approval are essential for the next steps in the process.
+                </mj-text>
+                <mj-text>
+                    Here are the details of the application:
+                </mj-text>
+                <mj-text>
+                    <ul>
+                        <li>
+                            Applicant Name: ${applicantName} <br/>
+                        </li>
+                        <li>
+                            Application ID: ${id} <br/>
+                        </li>
+                        <li>
+                            Submission Date: ${submittedDate} <br/>
+                        </li>
+                    </ul>
+                </mj-text>
+                <mj-text css-class="bold">
+                    Action Required
+                </mj-text>
+                <mj-text>
+                    Please carefully review the application. There are two ways to do so, you can review using the <a href="${ui}" target="_blank" rel="nofollow">DACO portal </a>, or download the application PDF from the 'Sign & Submit' page. 
+                </mj-text>
+                <mj-text>
+                    If you are satisfied with the details, proceed to the 'ign & Submit' page' provide your e-signature, and click on the 'Submit Application' button to complete the review. The application will be forwarded to the Data Access Committee for further review. The applicant will receive an email notification regarding your submission.
+                </mj-text>
+                <mj-text>
+                    If you are not satisfied with the application and would like to request revisions, click the 'Request Revisions' button. A window will open where you can enter requests. When you are done, click the 'Send Request' button to complete the review. Please do not submit the application if you requested revisions. The applicant will receive an email notification about your requests and comments, and they will update the application based on your feedback. DACO will notify you when the application is re-submitted for your review. 
+                </mj-text>
+                <mj-text>
+                    Please access the application through <a href="${ui}" target="_blank" rel="nofollow">DACO portal </a> at your earliest convenience. If you encounter any issues or have questions, feel free to reach out, and we would be happy to assist.
```

Comment:
extra space will show here with a extra _, also we should capitalize portal?
```suggestion
                    Please access the application through <a href="${ui}" target="_blank" rel="nofollow">DACO Portal</a> at your earliest convenience. If you encounter any issues or have questions, feel free to reach out, and we would be happy to assist.
```